### PR TITLE
Add a new Bolt metric `<prefix>.dbms.bolt.worker_thread_bound_time`

### PR DESCRIPTION
**Pull request by NataliaIvakina**
depends on the https://github.com/neo-technology/neo4j/pull/25609 and https://github.com/neo-technology/neo4j/pull/25786

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1660.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1660.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Comment by NataliaIvakina** [comment]
@dotStart, you don't define the metric's type here. Is it done intentionally? I suppose it has to be counter.

**Comment by dotStart** [comment]
It is a histogram consisting of representative samples. I'll amend the annotations to reflect that.

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 1]
```diff
@@ -92,6 +92,7 @@ By default, database metrics include:
 |<prefix>.dbms.bolt.messages_failed|The total number of messages that have failed while processing. A high number of failures may indicate an issue with the server and further investigation of the logs is recommended. (counter)
 |<prefix>.dbms.bolt.accumulated_queue_time|(unsupported feature) When `internal.server.bolt.thread_pool_queue_size` is enabled,  the total time in milliseconds that a Bolt message waits in the processing queue before a Bolt worker thread becomes available to process it. Sharp increases in this value indicate that the server is running at capacity. If `internal.server.bolt.thread_pool_queue_size` is disabled, the value should be `0`, meaning that messages are directly handed off to worker threads. (counter)
 |<prefix>.dbms.bolt.accumulated_processing_time|The total amount of time in milliseconds that worker threads have been processing messages. Useful for monitoring load via Bolt drivers in combination with other metrics. (counter)
+|<prefix>.dbms.bolt.worker_thread_bound_time|label:new[Introduced in 5.21]The amount of time in milliseconds that worker threads spent bound to a given connection
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:95]:
```suggestion
|<prefix>.dbms.bolt.worker_thread_bound_time|label:new[Introduced in 5.21]The amount of time in milliseconds that worker threads spent bound to a given connection. (histogram)
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1660.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1660.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Comment by NataliaIvakina** [comment]
@dotStart, I've updated the metric's description. Is it ready to be merged?

**Pull request merged**